### DISCUSSION TITLE
New Carousel Type (carouselType)

### DISCUSSION THREAD

**Discussion by ghost** [ImaginativeShohag/Why-Not-Image-Carousel#77] [Q&A] [answered]
Hello friend @ImaginativeShohag, I am using your dependency in a personal project and I need to find a way to reduce the margins of the back images, to make the following type of carousel I am using the following properties:
         
autoWidthFixing = true
infiniteCarousel = true
scaleOnScroll = true
scalingFactor = 0.25f
imageScaleType = ImageView.ScaleType.CENTER_CROP
carouselGravity = CarouselGravity.CENTER

Objetive:
![image](https://user-images.githubusercontent.com/48499815/152886278-9242343c-d725-48cc-9e78-b7056c8d0a55.png)

![image](https://user-images.githubusercontent.com/48499815/152886655-e48086dd-4f73-4e7d-88f8-2d629a2c5088.png)

**Comment by ImaginativeShohag** [comment, marked as answer]
In the current implementation, you can't have this type of overlapping view. You can play with the [utils](https://github.com/ImaginativeShohag/Why-Not-Image-Carousel/tree/master/whynotimagecarousel/src/main/java/org/imaginativeworld/whynotimagecarousel/utils) to change the behavior. :)

**Comment by ghost** [comment]
Good day thank you very much @ImaginativeShohag 
when i import the whynotimagecarousel project i get the following error in android-studio:
![image](https://user-images.githubusercontent.com/48499815/153002607-ea3335c6-3414-4bd3-b495-f4d019cb3623.png)

![image](https://user-images.githubusercontent.com/48499815/153002689-b17f0610-c312-41e6-8223-28efcadab5a1.png)

I have tried to import the project in intellij and I can not get it to start, how can I access the utilities, can I edit the project code? Thank you very much friend

**Reply by ImaginativeShohag** [reply]
Just open the full clone folder to Android Studio. Not just `whynotimagecarousel` folder.

And commented out [this](https://github.com/ImaginativeShohag/Why-Not-Image-Carousel/blob/45427d799a26a92ce6327046762edb6b9475ddbe/whynotimagecarousel/build.gradle#L6) line and sync gradle. Now the project should run.

**Reply by ghost** [reply]
Perfect, very kind, thank you very much, now if I start the project, I'll try ;)
